docs/UserGuide.md

```diff
@@ -50,19 +50,27 @@ Paidlancers is a **desktop app for keeping track of your freelancing events, opt
 
 ## Features
 
-### Add a Contact
-Adding a contact : `newcontact`
-Adding a contact to contacts
-Format: newcontact NAME NUMBER
-Creates a new contact with specified NAME and NUMBER
+### Add a Contact : `newcontact`
+
+Adding a contact to contacts.
+
+Format: `newcontact /n NAME /no NUMBER`
+
+Creates a new contact with specified `NAME` and `NUMBER`
+
 Example:
-newcontact Deborah Tan /num 91234567
+- `newcontact /n Deborah Tan /no 91234567`
+
+- `newcontact /n Bala Krisnan /no 98765432`
+
+
+### View all contacts : `listcontact`
+
+Displays all contacts saved in a list.
 
+Format: `listcontact`
```

Comment:
Thank you for working on this @bryansendeavour, however I think this overlaps with #35 that Gloria has worked on, since #35 is a superset of this, recommended to close the PR.